docs/src/pinn/neural_adapter.md

```diff
@@ -0,0 +1,252 @@
+## Neural Adapter
+
+Using the example of 2D Poisson equation, it is shown how, using method neural_adapter, to retrain the prediction of one neural network to another.
+
+![image](https://user-images.githubusercontent.com/12683885/127149639-c2a8066f-9a25-4889-b313-5d4403567300.png)
+
+```julia
+using NeuralPDE, Flux, ModelingToolkit, GalacticOptim, Optim, DiffEqFlux, DiffEqBase
+import ModelingToolkit: Interval, infimum, supremum
+
+@parameters x y
+@variables u(..)
+Dxx = Differential(x)^2
+Dyy = Differential(y)^2
+
+# 2D PDE
+eq  = Dxx(u(x,y)) + Dyy(u(x,y)) ~ -sin(pi*x)*sin(pi*y)
+
+# Initial and boundary conditions
+bcs = [u(0,y) ~ 0.0, u(1,y) ~ -sin(pi*1)*sin(pi*y),
+       u(x,0) ~ 0.0, u(x,1) ~ -sin(pi*x)*sin(pi*1)]
+# Space and time domains
+domains = [x ∈ Interval(0.0,1.0),
+           y ∈ Interval(0.0,1.0)]
+quadrature_strategy = NeuralPDE.QuadratureTraining(reltol=1e-2,abstol=1e-2,
+                                                   maxiters =50, batch=100)
+inner = 8
+af = Flux.tanh
+chain1 = Chain(Dense(2,inner,af),
+               Dense(inner,inner,af),
+               Dense(inner,1))
+
+initθ = Float64.(DiffEqFlux.initial_params(chain1))
+discretization = NeuralPDE.PhysicsInformedNN(chain1,
+                                             quadrature_strategy;
+                                             init_params = initθ)
+
+pde_system = PDESystem(eq,bcs,domains,[x,y],[u])
+prob = NeuralPDE.discretize(pde_system,discretization)
+sym_prob = NeuralPDE.symbolic_discretize(pde_system,discretization)
+
+res = GalacticOptim.solve(prob, BFGS();  maxiters=2000)
+phi = discretization.phi
+
+inner_ = 12
+af = Flux.tanh
+chain2 = FastChain(FastDense(2,inner_,af),
+                   FastDense(inner_,inner_,af),
+                   FastDense(inner_,inner_,af),
+                   FastDense(inner_,1))
+
+initθ2 =Float64.(DiffEqFlux.initial_params(chain2))
+
+function loss(cord,θ)
+    chain2(cord,θ) .- phi(cord,res.minimizer)
+end
+
+strategy = NeuralPDE.GridTraining(0.02)
+
+prob_ = NeuralPDE.neural_adapter(loss, initθ2, pde_system, strategy)
```

Comment:
why is this not using the params from `res`?

Reply:
it is using in 54th line 
```julia
function loss(cord,θ)
    chain2(cord,θ) .- phi(cord,res.minimizer)
end
```